driver-core/src/main/com/mongodb/connection/ConnectionDescription.java

```diff
@@ -86,16 +87,35 @@ public ConnectionDescription(final ConnectionId connectionId, final ServerVersio
     public ConnectionDescription(final ConnectionId connectionId, final ServerVersion serverVersion,
                                  final ServerType serverType, final int maxBatchCount, final int maxDocumentSize,
                                  final int maxMessageSize, final List<String> compressors) {
+        this(connectionId, serverVersion, serverType, maxBatchCount, maxDocumentSize, maxMessageSize, compressors, new BsonTimestamp());
+    }
+
+    /**
+     * Construct an instance.
+     *
+     * @param connectionId    the connection id
+     * @param serverVersion   the server version
+     * @param serverType      the server type
+     * @param maxBatchCount   the max batch count
+     * @param maxDocumentSize the max document size in bytes
+     * @param maxMessageSize  the max message size in bytes
+     * @param compressors     the available compressors on the connection
+     * @param operationTime   the last operation time
```

Comment:
Added the last seen operation time to the connection description - so it could be used when creating the changestream command.

Reply:
There's been a lot of chatter about this in #drivers and other Slack channels.  The consensus seems to be that we need to ensure a more recent operation time.  

My conclusion so far is that the driver should get it from the ClusterClock.    For Java driver that would mean adding a new property to Cluster that exposes BaseCluster.clusterClock.getCurrent(), and then adding a method to ReadBinding to expose it to ReadOperations.

Reply:
Done